code/modules/client/preferences.dm

```diff
@@ -1103,7 +1109,7 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 								continue
 							if(gender == FEMALE && S.gender == MALE)
 								continue
-							if( !(species in S.species_allowed))
+							if(!(species in S.species_allowed))

```

Comment:
That's what I figured but I wasn't entirely sure until I tested it, I'm glad someone else is able to confirm this haha